cmd/podman/containers/create.go

```diff
@@ -260,10 +261,23 @@ func createInit(c *cobra.Command) error {
 }
 
 func pullImage(imageName string) (string, error) {
-	pullPolicy, err := config.ValidatePullPolicy(cliVals.Pull)
+
+	pullPolicy, err := config.ParsePullPolicy(cliVals.Pull)
+
 	if err != nil {
 		return "", err
 	}
+	//Added runOpts2 to the Struct containerRunOptionsWrapper for `entities.ContainerRunlabelOptions`
+	//As SkipTLSVerify comes from ContainerRunlabelOptions
+	//If Flag changed for TLS Verification
+
+	//	if cliVals.TlsVerify == true {
+	//if types.NewOptionalBool(cliVals.TlsVerify) == types.OptionalBoolTrue {
+
+	//	&pullReport.SkipTLSVerify = types.NewOptionalBool(!cliVals.TlsVerify)
+	fmt.Println("----tls-verify---:", cliVals.TlsVerify)
+
+	//entities.ImagePullReport.SkipTLSVerify
```

Comment:
That should be removed.

Reply:
Done, removed, it was for a check purpose.